airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -665,17 +669,99 @@ class Commits(IncrementalGithubStream):
         "committer",
     )
 
-    def transform(self, record: MutableMapping[str, Any], repository: str = None, **kwargs) -> MutableMapping[str, Any]:
+    def __init__(self, branches_to_pull: Mapping[str, List[str]], default_branches: Mapping[str, str], **kwargs):
+        super().__init__(**kwargs)
+        self.branches_to_pull = branches_to_pull
+        self.default_branches = default_branches
+
+    """
+    Pull commits from each branch of each repository, tracking state for each branch
```

Comment:
Should we move this comment to the above docstring? Which starts from `API docs: https...`.

Reply:
Also could you please update the link to the API documentation for `Commits` stream because current link is incorrect (in docstring for class and in `github.md` file). Here is a corect one - https://docs.github.com/en/rest/reference/repos#list-commits.